felix.go

```diff
@@ -401,13 +405,12 @@ configRetry:
 			}
 		}()
 
-		go usagerep.PeriodicallyReportUsage(
+		usageRep := usagerep.New(
 			24*time.Hour,
-			configParams.ClusterGUID,
-			configParams.ClusterType,
-			configParams.CalicoVersion,
 			statsChanOut,
+			configUpdChan,
```

Comment:
Is it possible that Felix reads ClusterType etc. in its initial read from the datastore (`loadConfigFromDatastore`), and that those settings *never* change after that, and so there are never any updates through `configUpdChan`?

Would the usagerep still report correctly, in that case?

Reply:
The calculation graph doesn't report deltas from the initial load of config, it reports all the config form the initial snapshot so it should send the initial update.

Reply:
Ah right - fine.